### PR TITLE
IGNITE-7647

### PR DESCRIPTION
**Pull request by vveider**

### CODE REVIEW

File: packaging/rpm/SPECS/apache-ignite.spec [hunk 1]
```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
...
+Provides:         %{name}-core
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameCore}
+Apache Ignite's core files and libs, necessary for node operation
+
 
+%package -n %{nameLibs}
+Summary:          Apache Ignite optional libs
```
Comment (alamar) [inline packaging/rpm/SPECS/apache-ignite.spec]: Why no word "optional" in package name then?

File: packaging/rpm/SPECS/apache-ignite.spec [hunk 2]
```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
...
+Provides:         %{name}-libs
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameLibs}
+Apache Ignite's optional libs and integrations
+
+
+%package -n %{nameBenchmarks}
+Summary:          Apache Ignite benchmarks
```
Comment (alamar) [inline packaging/rpm/SPECS/apache-ignite.spec]: Are they really runnable? Did you try it?

File: packaging/rpm/SPECS/apache-ignite.spec [hunk 3]
```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
...
+Provides:         %{name}-examples
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
+%description -n %{nameExamples}
+Apache Ignite's integration and usage examples
+
+
+%package -n %{nameDocs}
+Summary:          Apache Ignite documentation
```
Comment (alamar) [inline packaging/rpm/SPECS/apache-ignite.spec]: Only saw scaladocs there, no javadocs. Is my build process broken?

File: packaging/rpm/SPECS/apache-ignite.spec [hunk 4]
```diff
@@ -25,14 +34,113 @@ BuildArch:        noarch
...
+Summary:          Apache Ignite C++ platform
+Version:          %{version}
+Group:            Development/System
+License:          ASL 2.0
+URL:              https://ignite.apache.org
+Obsoletes:        apache-ignite < 2.5.0
+Provides:         %{name}-cpp
+AutoReq:          no
+AutoProv:         no
+BuildArch:        noarch
```
Comment (alamar) [inline packaging/rpm/SPECS/apache-ignite.spec]: That's a problem here. What the platform exactly they are for? Is there native code?

File: packaging/rpm/SPECS/apache-ignite.spec (hunk 1 above)
Comment (vveider) [reply packaging/rpm/SPECS/apache-ignite.spec]: Optional is meant "by default" when you have package that is not required for the core.

File: packaging/rpm/SPECS/apache-ignite.spec (hunk 2 above)
Comment (vveider) [reply packaging/rpm/SPECS/apache-ignite.spec]: No, I have no knowledge about benchmarks, but theirs layout (except some libs mapping) did not changed from 2.4.0 package.

File: packaging/rpm/SPECS/apache-ignite.spec (hunk 3 above)
Comment (vveider) [reply packaging/rpm/SPECS/apache-ignite.spec]: If built from correct sources, both `scaladoc` and `javadoc` exist in package. See (this build!)[https://ci.ignite.apache.org/viewLog.html?buildId=1169369&buildTypeId=Releases_NightlyRelease_ApacheIgniteNightlyReleaseAssembleRpmPackages&tab=artifacts] for example.

File: packaging/rpm/SPECS/apache-ignite.spec (hunk 4 above)
Comment (vveider) [reply packaging/rpm/SPECS/apache-ignite.spec]: There are sources and headers which require preliminary compilation

**Comment by dspavlov** [comment]
IGNITE-7647 | Closed
-- | --

**Pull request closed without merging**